### PR TITLE
fix running internal tox test runs from path with spaces

### PR DESCRIPTION
**Pull request by jurko-gospodnetic**
Fixed running internal tox test runs from path with spaces by adding quotes around command arguments that may contain spaces in them.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -89,8 +89,8 @@ exclude_lines = if __name__ == ["']__main__["']:
 
 [coverage:paths]
 source = tox
-         {toxworkdir}/*/lib/python*/site-packages/tox
-         {toxworkdir}/*/Lib/site-packages/tox
+         {toxworkdir}/*/lib/python*/site-packages/tox"
```
Comment (RonnyPfannschmidt) [inline tox.ini]: at first glance the ones in the front seem to be missing

**Comment by RonnyPfannschmidt** [comment]
👍

**Comment by gaborbernat** [comment]
@jurko-gospodnetic rebase and the CI now should be good

File: tox.ini (hunk 1 above)
Comment (jurko-gospodnetic) [reply tox.ini]: doh.... sorry... those seem to have been left in by mistake - nothing needs to be quoted there - fixing it now

**Comment by jurko-gospodnetic** [comment]
Travis failure is for a lint issue introduced recently on `master` and fixed with https://github.com/tox-dev/tox/pull/718.

**Pull request merged**